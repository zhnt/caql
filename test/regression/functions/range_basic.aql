// Basic range test
let r = range(1, 4, 1)
print(r)

