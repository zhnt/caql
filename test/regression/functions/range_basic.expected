range(1, 4, 1)

